app/src/main/java/edu/uc/app/parkfinder/dto/Park.kt

```diff
@@ -2,6 +2,9 @@ package edu.uc.app.parkfinder.dto
 
 import com.google.gson.annotations.SerializedName
 
-data class Park(@SerializedName("Code") var code: String, @SerializedName("Name") var name: String){
+data class Park(
+    @SerializedName("Park Code") var code: String,
+    @SerializedName("Park Name") var name: String
```

Comment:
Changed to fit the kotlin style guide https://developer.android.com/kotlin/style-guide
Serialized names changed to be more understandable